ako-gateway-api/nodes/dequeue_ingestion.go

```diff
@@ -62,6 +62,11 @@ func DequeueIngestion(key string, fullsync bool) {
 			utils.AviLog.Errorf("key: %s, msg: no model found: %s", key, modelName)
 			continue
 		}
+		if objType == utils.Secret {
+			handleGateway(parentNs, parentName, fullsync, key)
```

Comment:
In the next release, we should revisit this logic and call the method that actually does the secret handling.

Reply:
Sure. Noted

Reply:
With this logic, we will work on all secret associated with that GW. Can we optimize this?

Reply:
Optimised it and added calling for the method that does only Secret handling